.github/workflows/release.yaml

```diff
@@ -285,16 +298,26 @@ jobs:
   linux-build:
     strategy:
       matrix:
+        # Keep ubuntu-latest for general builds, add ubuntu-24.04-arm
+        os: [ ubuntu-latest ]
```

Comment:
Similarly, `linux` and `linux-arm64` are GitHub actions runners. Please don't change it arbitrarily